cmd/kubeadm/app/phases/selfhosting/selfhosting_test.go

```diff
@@ -23,8 +23,9 @@ import (
 	"os"
 	"testing"
 
-	"github.com/ghodss/yaml"
+	"k8s.io/api/extensions/v1beta1"
```

Comment:
everything here has now moved to using `apps/v1beta2` as you rebase

Reply:
we often import this as the group name; like `extensions` or `apps` respectively